src/styles/utils/_vars.scss

```diff
@@ -1 +1,26 @@
 $c-gray: #eee;
+$c-light-black: #292929;
+$c-white: #fff;
+
+$smartphone-width: 320px;
+$tablet-width: 744px;
+$desktop-width: 1260px;
+
+$smartphone-horizontal-padding: 20px;
+$tablet-horizontal-padding: 72px;
+$desktop-horizontal-padding: 120px;
+
+$tablet-grid-columns: 6;
+$tablet-grid-coumn-width: 80px;
+$desktop-grid-columns: 12;
+$desktop-grid-coumn-width: 63px;
```

Comment:
There is a typo in the variable name, change '$desktop-grid-coumn-width' to '$desktop-grid-column-width'.